resources/assets/coffee/react/beatmap-discussions/header.coffee

```diff
@@ -121,6 +121,11 @@ class BeatmapDiscussions.Header extends React.PureComponent
               beatmap: @props.currentBeatmap
 
 
+  discussionUsers: =>
```

Comment:
How about `discussionStarters` or something to be more descriptive?

Also not sure if it's better to do this once-off when discussions are loaded as opposed to each render?